index.php

```diff
@@ -52,8 +52,13 @@
 <noscript><p><img src="//stats.wikimedia.de/piwik.php?idsite=3&amp;rec=1" style="border:0;" alt=""/></p></noscript>
 <!-- End Matomo Code -->
 
-<script type="application/javascript" src="https://bruce.wikipedia.de/<?php echo $urlBanner; ?>"></script>
+<script type="application/javascript" id="bruceScriptTag" data-banner-src="https://bruce.wikipedia.de/<?php echo $urlBanner; ?>"></script>
 <script>
+    var bruceScriptTag = $('#bruceScriptTag');
+    var bannerUrl = bruceScriptTag.data('banner-src');
+    var vWidthParam = "?vWidth=" + window.innerWidth + '"';
+    bruceScriptTag.append('<script src="' + bannerUrl + vWidthParam + '><\/script>');
```

Comment:
jQuery append puts the new content into the selected element, so won't this end up nesting the script tags?